pkg/placement/placement.go

```diff
@@ -269,7 +271,7 @@ func (p *Service) ReportDaprStatus(stream placementv1pb.Placement_ReportDaprStat
 		case nil:
 
 			if clientID != nil && req.GetId() != clientID.AppID() {
-				return status.Errorf(codes.PermissionDenied, "client ID %s is not allowed", req.GetId())
+				return apiErrors.PlacementServicePermissionDenied(fmt.Sprintf("Failed wih code %s. Client ID %s is not allowed", codes.PermissionDenied, req.GetId()))
```

Comment:
Same comment as above, let's follow the go styling guide and start all error messages with lowercase letters. Also, no need to wrap the error message with "Failed with code"